lib/validate.ex

```diff
@@ -31,4 +31,22 @@ defmodule Fields.Validate do
 
     Regex.match?(regex, postcode)
   end
+
+  @doc """
+  Validate an address.
+  Currently just validates that some input has been given.
+  """
+  def address(address) do
+    String.length(address) > 0
+  end
+
+  @doc """
+  Validates the format of a UK phone number.
```

Comment:
perhaps we should have a `phone_number_uk/1` or `phone_number/2` 
invoked as `Validate.phone_number(+44789654321, "UK")` ?

We can _ship_ with this for CS but will need to consider the _future_.
Let's discuss future requirements in the issue. :+1: